src/main/java/com/stripe/model/Charge.java

```diff
@@ -405,46 +459,76 @@ public Dispute closeDispute(RequestOptions options)
         Dispute.class, options);
   }
 
+  /**
+   * Close the charge's dispute.
+   *
+   * @deprecated Use the {@link Dispute#close(RequestOptions)} method instead.
+   *     This method will be removed in the next major version.
+   */
   @Deprecated
   public Dispute closeDispute(String apiKey)
       throws AuthenticationException, InvalidRequestException,
       APIConnectionException, CardException, APIException {
     return closeDispute(RequestOptions.builder().setApiKey(apiKey).build());
   }
+  // </editor-fold>
 
+  // <editor-fold desc="create">
+  /**
+   * Create a charge.
+   */
   public static Charge create(Map<String, Object> params)
       throws AuthenticationException, InvalidRequestException,
       APIConnectionException, CardException, APIException {
     return create(params, (RequestOptions) null);
   }
 
+  /**
+   * Create a charge.
+   */
   public static Charge create(Map<String, Object> params, RequestOptions options)
       throws AuthenticationException, InvalidRequestException,
       APIConnectionException, CardException, APIException {
     return request(RequestMethod.POST, classURL(Charge.class), params, Charge.class, options);
   }
 
+  /**
+   * Create a charge.
+   *
+   * @deprecated Use the {@link #create(Map, RequestOptions)} method instead.
+   *     This method will be removed in the next major version.
+   */
   @Deprecated
   public static Charge create(Map<String, Object> params, String apiKey)
       throws AuthenticationException, InvalidRequestException,
       APIConnectionException, CardException, APIException {
     return create(params, RequestOptions.builder().setApiKey(apiKey).build());
   }
+  // </editor-fold>
 
+  // <editor-fold desc="list">
+  /**
+   * List all charges.
+   */
   public static ChargeCollection list(Map<String, Object> params)
       throws AuthenticationException, InvalidRequestException,
       APIConnectionException, CardException, APIException {
     return list(params, null);
   }
 
+  /**
+   * List all charges.
+   */
   public static ChargeCollection list(Map<String, Object> params, RequestOptions options)
       throws AuthenticationException, InvalidRequestException,
       APIConnectionException, CardException, APIException {
     return requestCollection(classURL(Charge.class), params, ChargeCollection.class, options);
   }
+  // </editor-fold>
 
+  // <editor-fold desc="markFraudulent">
   /**
-   * Convenience method to mark a given charge as fraudulent.
+   * Mark the charge as fraudulent.
    */
   public Charge markFraudulent(RequestOptions options)
```

Comment:
I made an exception for those two because it is mildly annoying to do this in Java (you have to instantiate two maps just for this) so I think a convenience method is warranted. I don't feel too strongly about it though.